src/renderer/components/ContextMenu.tsx

```diff
@@ -1,15 +1,24 @@
 import React, { useState, useEffect, useLayoutEffect, useContext } from 'react'
+import classNames from 'classnames'
 import { useRef } from 'react'
 import { ScreenContext, unwrapContext } from '../contexts'
 
-export type ContextMenuItem = { label: string; action: () => void }
+export type ContextMenuItem = {
+  label: string
+  action: () => void
+  subitems?: (ContextMenuItem | undefined)[]
+}
```

Comment:
I think it makes sense to make `action` and `subitems` mutually exclusive:
I did not test it, but sth like this should do the trick
```suggestion
export type ContextMenuItem = 
{ label: string } & (
 {action: () => void} | 
 {subitems?: (ContextMenuItem | undefined)[]}
)
```